Bot.Core/Game.cs

```diff
@@ -9,17 +9,19 @@ public class Game : IGame
 		public TownKey TownKey { get; }
 		public IReadOnlyCollection<IMember> Storytellers => m_storytellers;
 
-		private readonly List<IMember> m_storytellers = new();
+		private readonly HashSet<IMember> m_storytellers = new();
```

Comment:
You are actually setting this in the constructor already so you don't need to make a new one here